root/static/scripts/common/components/FingerprintTable.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React, {useEffect, useState} from 'react';
+
+import loopParity from '../../../../utility/loopParity';
+import hydrate from '../../../../utility/hydrate';
+
+function orderTracks(tracks) {
+  tracks.sort(function (a, b) {
+    if (a.disabled && !b.disabled) {
+      return 1;
+    }
+    if (!a.disabled && b.disabled) {
+      return -1;
+    }
+    if (a.id > b.id) {
+      return 1;
+    }
+    if (a.id < b.id) {
+      return -1;
+    }
+    return 0;
+  });
+  return tracks;
+}
+
+const FingerprintTable = ({recording}: {recording: RecordingT}) => {
+  const [tracks, setTracks] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  // We ensure fetch only runs client-side since it's not in node
+  useEffect(() => {
+    fetch(
+      '//api.acoustid.org/v2/track/list_by_mbid' +
+      `?format=json&disabled=1&jsoncallback=?&mbid=${recording.gid}`,
+    ).then(
+      function (response) {
+        return response.json();
+      },
+    ).then(
+      function (data) {
+        const orderedTracks = orderTracks(data.tracks);
```

Comment:
Something like `data.tracks.sort(compareTracks)` here is what I meant to suggest, with `compareTracks` being the function that accepts `(a, b)`. BTW, `.sort` on arrays sorts in-place, so you don't have to assign it to anything.